### ISSUE TITLE
Blacklist

### ISSUE THREAD

**Issue by JavierCasillas13** [Null3rror/OpenVPN-over-Proxy#6]
I want to congratulate you on your excellent work but I have a question, how can I make the blakclist work since it doesn't lock the lists?

In the blacklistIPs.txt file I add the addresses that I don't want but it doesn't seem to work, I also added the 
http_access deny blacklistSrc command but it still doesn't work, do you know why?

**Comment by Null3rror** [comment]
Hey Javier,
Did you restart squid? incase you didn't do this `service squid restart`

**Comment by JavierCasillas13** [comment]
Yes, restart the service and I made some small changes in the squid.conf

[image: imagen.png]

this is the blacklist file

[image: imagen.png]
these are the iptables rules
[image: imagen.png]
I take facebok and instagram as an example to block some web pages, but
when I connect through the vpn it seems to not block the sites that I
mention, do you have any idea why?

**Comment by Null3rror** [comment]
I can not see your pictures
